client/app/js/css/style.css

```diff
@@ -95,6 +95,12 @@ input {
     width: 300px;
     margin: 150px auto 0px auto;
 }
+#messageboard {
```

Comment:
I am considering the folder structure of this. You can have the css folder not nested under the js folder